analysis.R

```diff
@@ -1,138 +1,129 @@
-###################
-##  12/27/21  ##
```

Comment:
Just personal preference but in my experience (a) git will keep track of dates for you and (b) I'll often forget to edit the date at the top of the document which just makes things confusing